blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,266 @@
+[these instructions](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), and to set up your
+SSH key with GitLab, follow [these instructions](https://docs.gitlab.com/ee/user/ssh.html#add-an-ssh-key-to-your-gitlab-account).
+
+#### GoLand
+
+Most parts of Knative are developed in the Go programming language, so you will need to have this language installed to be able to develop Knative. There are some
+parts of Knative written in other languages, but we will be discussing those in __Post 7__ of this blog series (coming soon!). To install the go programming language,
+follow the instructions [in this article](https://go.dev/doc/install).
+
+#### An Editor 
+
+In order to edit the code, you will need to have an editor installed. The most popular and free editor used these days is [VSCode](https://code.visualstudio.com/), 
+although the paid Jetbrains editors such as [GoLand](https://www.jetbrains.com/go/), and [IntelliJ IDEA](https://www.jetbrains.com/idea/) are also very powerful.
+Some also prefer to go old school and use [Vim](https://www.vim.org/), [Emacs](https://www.gnu.org/software/emacs/), or the newer fork of Vim, [Neovim](https://neovim.io/)
+(which actually has all the language support that VSCode has if you are willing to configure it yourself). While your choise of editor doesn't impact the quality of code
+which you will produce, it is crucial that you choose one and become comfortable working in it as much of your productivity will come from your ability to use your editor.
+
+### Running Knative Locally
+
+Now that you have verified that you have all the prerequisites, let's get started setting up your local system to run Knative!
+
+#### Docker/Podman Setup
+
+Knative runs in a Kubernetes Cluster, which means it runs as a set of Containers. You can think of a container as something which both contains your code and all the
+dependencies needed to run your code, as well as something which runs your code with those packaged dependencies. The two most common tools used to build and run containers
+are __Docker__ and __Podman__, so we will discuss setting both up here. However, do note that we think that __using Docker__ is easier due to its direct integration with a
+container registry. As such, we recommend that you use Docker.
+
+##### Option 1: Setting up Docker
+
+The first step to setting up Docker is to install it. The easiest way to do this is to [install Docker Desktop](https://docs.docker.com/get-docker/), although you only need
+the Docker Engine for Knative so you can also just [install the Docker Engine](https://docs.docker.com/engine/install/). Note: if you plan on just installing the Docker Engine,
+make sure that you complete the post-install steps.
+
+Once you have Docker installed on your system, you will need to log into a docker account in order to connect to a docker registry later (this is a common step when building
```

Comment:
That's probably one of the easiest ways, but can lead to rate limits. Wondering, if we should mention other solutions too (like Quay / local registries). Just that the user is aware, that it's not required for them to create a Docker account :thinking: :shrug: